src/Laserfiche.Repository.Api.Client.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Laserfiche.Api.Client.Core" Version="1.2.3" />
+    <PackageReference Include="Laserfiche.Api.Client.Core" Version="1.3.0-beta-3575746395" />
```

Comment:
Currently using preview package. When corresponding PR is merged, will create 1.3.0 package and update this